CMakeLists.txt

```diff
@@ -396,6 +413,13 @@ endif() # SINGULARITY_USE_FORTRAN
 target_include_directories(
   singularity-eos PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
                          $<INSTALL_INTERFACE:include>)
+target_include_directories(
+  singularity-eos PUBLIC $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/generated>)
+
+get_property(plugin_include_paths GLOBAL PROPERTY PLUGIN_INCLUDE_PATHS)
+foreach(path ${plugin_include_paths})
+  target_include_directories(singularity-eos PUBLIC $<BUILD_INTERFACE:${path}>)
+endforeach()
 
```

Comment:
Shouldn't we have an install interface here? I think it'd just be the same as the `$<INSTALL_INTERFACE:include>`, but there's other scaffolding to manage (`INSTALL()` and `EXPORT()`)

Reply:
I just hooked it into your current install machinery. If you look above, I add headers to the install headers list object. And for the source files, they all get linked into a single library anyway. That additional `foreach` that you comment on here is just to make sure in-tree/submodule mode builds work.